templates/settings.php

```diff
@@ -237,6 +237,33 @@
 			<?php \do_settings_fields( 'activitypub', 'general' ); ?>
 			<?php \do_settings_fields( 'activitypub', 'server' ); ?>
 		</div>
+		<div class="box">
```

Comment:
should this be hidden when the constant is defined?

Reply:
good point!